src/app.ts

```diff
@@ -21,6 +21,8 @@ export interface WebhookInterface {
     url?: string;
     lambda_function?: string;
     event_types: string[];
+    channel_starts_with?: string;
+    channel_ends_with?: string;
```

Comment:
Would it be a better idea to have a separate section like `filter` or something? That would be easier to group things, I guess. 🤔

Reply:
Fine with me. Simply adding a `filter` and nest these 2 under or would you change the options themself to some other name too?

Reply:
I was thinking this is a good idea for more filters, not just the channel name that starts or ends with something. Making it useful in the future.